### PR TITLE
fix(ValidationMessage): adjust size and position of icon in validation

### PR DESCRIPTION
**Pull request by domarku**
- Adjust size and position of icon in the validation message below the input field

### Current state

The icon is aligned to the center of the text block and resizes with every new line of text.

Screenshot 2020-05-28 at 17 29 54

### Proposed solution

Screenshot 2020-05-28 at 17 21 20

## PR Checklist

- [x] I have read the relevant `readme.md` file(s)
- [x] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Storybook stories are added/updated/not required
- [x] Usage notes are added/updated/not required
- [x] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [x] Doesn't contain any sensitive information

### CODE REVIEW

File: packages/forma-36-react-components/src/components/ValidationMessage/ValidationMessage.css [hunk 1]
```diff
@@ -4,12 +4,13 @@
 
 .ValidationMessage {
   display: flex;
-  align-items: center;
 }
 
 .ValidationMessage__icon {
   margin-right: calc(1rem * (7 / var(--font-base-default)));
-  margin-bottom: calc(1rem * (1 / var(--font-base-default)));
+  margin-top: calc(1rem * (2 / var(--font-base-default)));
```
Comment (mshaaban0) [inline packages/forma-36-react-components/src/components/ValidationMessage/ValidationMessage.css:11]: Nice @domarku 🔥do you want to use one of the spacing values here instead?

Comment (domarku) [reply packages/forma-36-react-components/src/components/ValidationMessage/ValidationMessage.css]: 100% yes, thanks for the suggestion

Comment (domarku) [reply packages/forma-36-react-components/src/components/ValidationMessage/ValidationMessage.css]: Actually, come to think of it, when icons are put in context with text, it's good practice to use the base font size to set the relationship between the two. If the base font size changes, the chances are greater that it's still going to work, as opposed to using an external spacing variable. @m10l any thoughts?

File: packages/forma-36-react-components/src/components/ValidationMessage/ValidationMessage.css [hunk 2]
```diff
@@ -1,15 +1,17 @@
 @import 'settings/colors';
 @import 'settings/typography';
 @import 'settings/transitions';
+@import '@contentful/forma-36-tokens/dist/css/spacing.css';
```
Comment (m10l) [inline packages/forma-36-react-components/src/components/ValidationMessage/ValidationMessage.css:4]: This should work and will keep things more consistent -

```suggestion
@import 'settings/dimensions';
```

File: packages/forma-36-react-components/src/components/ValidationMessage/ValidationMessage.css (hunk 1 above)
Comment (m10l) [reply packages/forma-36-react-components/src/components/ValidationMessage/ValidationMessage.css]: No strong opinion from me, think there are good arguments for both

**Comment by domarku** [comment]
Ready to merge!

**Pull request merged**